internal/controllers/nodefeaturediscovery_reconciler.go

```diff
@@ -330,23 +330,25 @@ func (nfdh *nodeFeatureDiscoveryHelper) handlePrune(ctx context.Context, nfdInst
 		return false, fmt.Errorf("failed to get nfd-prune job: %w", err)
 	}
 
-	deleteJob := false
+	//deleteJob := false
```

Comment:
Why leave this line commented and not simply remove it?

Reply:
my bad, forgot to remove it, fixed